specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/types.json

```diff
@@ -247,6 +265,15 @@
       "type": "string",
       "x-ms-parameter-location": "method"
     },
+    "associatedTenantName": {
+      "in": "path",
+      "name": "associatedTenantName",
+      "description": "The ID that uniquely identifies a tenant.",
+      "required": true,
+      "type": "string",
+      "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$",
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
Shouldn't "top", "filter", etc." all be prefixed with "$" (e.g. "$filter") to be OData-compliant?